app/app.html

```diff
@@ -0,0 +1,17 @@
+<ion-menu id="leftMenu" [content]="content">
+
+  <ion-toolbar>
+    <ion-title>Pages</ion-title>
+  </ion-toolbar>
+
+  <ion-content>
+    <ion-list>
+      <button ion-item *ngFor="#p of pages" (click)="openPage(p)">
+        {{p.title}}
+      </button>
+    </ion-list>
+  </ion-content>
+
+</ion-menu>
+
+<ion-nav id="nav" [root]="rootPage" #content swipe-back-enabled="false"></ion-nav>
```

Comment:
By adding #content above, we are creating a local reference called content to this component, which is what the menu is using. So the menu will use the navigation controller as its main content. We’re also setting the root property here to rootPage which we have defined in the class definition.